src/api/repositories/elasticsearch/variants.go

```diff
@@ -148,6 +150,33 @@ func GetDocumentsContainerVariantOrSampleIdInPositionRange(cfg *models.Config, e
 			}})
 	}
 
+	if len(alleles) > 0 {
+		switch len(alleles) {
+		case 1:
+			// any allele can be present on either side of the pair
+			shouldMap = append(shouldMap, map[string]interface{}{
+				"query_string": map[string]interface{}{
+					"query": "sample.variation.alleles.left.keyword:" + alleles[0],
+				}})
+			shouldMap = append(shouldMap, map[string]interface{}{
+				"query_string": map[string]interface{}{
+					"query": "sample.variation.alleles.right.keyword:" + alleles[0],
+				}})
+			minimumShouldMatch = 1
+		case 2:
+			// treat as a left/right pair
+			shouldMap = append(shouldMap, map[string]interface{}{
+				"query_string": map[string]interface{}{
+					"query": "sample.variation.alleles.left.keyword:" + alleles[0],
+				}})
+			shouldMap = append(shouldMap, map[string]interface{}{
+				"query_string": map[string]interface{}{
+					"query": "sample.variation.alleles.right.keyword:" + alleles[1],
```

Comment:
order only really matters if phase set is also specified, otherwise it should probably explicitly NOT query in order, since people may be interested in e.g. `C/A` without caring about order

Reply:
I'm not sure yet how often I'd need to query both alleles, but if I did, I almost certainly wouldn't be interested in order. If order matters in search, I'd have to launch every query twice (one for A/C, and again for C/A)